File: 17-2-2025/Task3.js
let num = 44
function sumOf(num){
    let sum = 0
    while(num>0){
        let n = num%10
        sum = sum+n
        num = Math.floor(num/10) //4
    }
    if(sum%2==0){
        console.log("Even")
    }else{
        console.log("Odd")
    }
}
sumOf(num)

File: Week-3/task9/Task3.java
package task9;

import java.util.List;

public class Task3 {
	public static void main(String[] args) {
		List<Integer> numbers = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);

		int evenSum = numbers.stream().filter(n -> n % 2 == 0).mapToInt(Integer::intValue).sum();

		int oddSum = numbers.stream().filter(n -> n % 2 != 0).mapToInt(Integer::intValue).sum();

		System.out.println("Sum of Even Numbers: " + evenSum);
		System.out.println("Sum of Odd Numbers: " + oddSum);
	}
}
